knowledge_repo/app/templates/index-base.html

```diff
@@ -39,48 +39,32 @@
   <div class='col-md-3 col-md-offset-4'>
     {% set start = feed_params['start'] | default(0) %}
     {% set results = feed_params['results'] | default(10) %}
-    {% if start == 0 %}
-      {% set page = 1 %}
-    {% else %}
-      {% set page = start//results + 1 %}
-    {% endif %}
+    {% set page = 1 if (start == 0) else (start//results + 1) %}
+
     <div class="btn-group btn-sm btn-group-justified pagination-group" role="group">
-      {% if page == 1 %}
-        <a href="#" class="btn btn-default disabled">
-          <i class="glyphicon glyphicon-chevron-left"></i>
-        </a>
-      {% else %}
-        {% set prev_page = url_for(request.endpoint, start=(page-2)*results, results=results, filters=feed_params['filters']) %}
-        {% if request.endpoint == 'tag.render_tag_pages' %}
-          {% set prev_page = prev_page + "&tag=" + full_tag %}
-        {% endif %}
-        <a href="{{ prev_page }}" class="btn btn-default">
+      {% set href = "#" if (page == 1) else request.path %}
+        <a href="{{ href }}" data-page="{{ page }}"
+           class="btn btn-default {{ "disabled" if (page == 1)}} prev-btn">
           <i class="glyphicon glyphicon-chevron-left"></i>
         </a>
-      {% endif %}
+
       {% if page == 1 %}
         {% set page_nums = [page, page + 1, page + 2] %}
       {% else %}
         {% set page_nums = [page - 1, page, page + 1] %}
       {% endif %}
       {% for page_num in page_nums %}
-        {% set href = url_for(request.endpoint, start=(page_num - 1) * results, results=results, filters=feed_params["filters"]) %}
-          {% if request.endpoint == 'tag.render_tag_pages' %}
-            {% set href = href + "&tag=" + full_tag %}
-          {% endif %}
+        {% set href = request.path %}
```

Comment:
good call, I can use that below. The reason I can't use feed_params here (which is why I took out in the diff) was because when doing `url_for`, you have to individually pass in all the keys in `feed_params`. This is problematic because different pages have different feed params (ie, tag_pages have a `tag` param, but `/index` doesn't), so you get a complicated if statement. by doing it in jquery you eliminate this complicated logic.